bin/release.sh

```diff
@@ -111,7 +174,7 @@ publish() {
         echo "# " "${git_push_tags[@]}"
     else
         "${git_tag[@]}"
-        "${git_push_tags[@]}"
+        #"${git_push_tags[@]}"
```

Comment:
why is this commented out? should we delete it? or, do we need to put it back so that the tag is actually pushed?

Reply:
Updated: I moved the push to a separate step after publish, with it's own confirmation.